docs/example_figure.py

```diff
@@ -0,0 +1,86 @@
+from matplotlib import pyplot as plt
+from matplotlib import colors as colors
+import numpy as np
+from astropy import units as u
+from screens.fields import dynamic_field
+from screens.visualization import make_sketch
+
+
+def gen_magnifs(th):
+
+    # Complex magnifications amplitudes of the scattering points
+    sig = 1.5*u.mas
+    a = 0.3*np.exp(-0.5*(th/sig)**2).to_value(1)
+    np.random.seed(2)
+    magnifs = a * np.random.normal(size=(th.size, 2)).view('c16').squeeze(-1)
+    magnifs[np.where(th == 0)] = 1
+    magnifs /= np.sqrt((np.abs(magnifs)**2).sum())
+
+    return magnifs
+
+
+def example_figs(th_par, magnifs,
+                 d_eff=2.*u.kpc, mu_eff=50.*u.mas/u.yr, fobs=330.*u.MHz,
+                 delta_f=1.*u.MHz, delta_t=20.*u.minute, nf=201, nt=101,
+                 **kwargs):
+
+    # Observation frequencies and times
+    f = fobs + np.linspace(-0.5*delta_f, 0.5*delta_f, nf)
+    t = np.linspace(-0.5*delta_t, 0.5*delta_t, nt)[:, np.newaxis]
+
+    # Create dynamic wavefield and dynamic spectrum
+    th_perp = np.zeros_like(th_par)
+    dynwave = dynamic_field(th_par, th_perp, magnifs, d_eff, mu_eff, f, t)
+    axes = tuple(range(0, dynwave.ndim-2))
+    dynwave = dynwave[...].sum(axes)
+    dynspec = np.abs(dynwave)**2
+
+    # Make conjugate and secondary spectrum
+    npad = 1
+    dspad = np.pad(dynspec, ((0, npad*nt), (0, npad*nf)),
+                   mode='constant', constant_values=np.mean(dynspec))
+    conspec = np.fft.fft2(dspad)
+    conspec /= conspec[0, 0]
+    tau = np.fft.fftfreq(dspad.shape[1], f[1] - f[0]).to(u.us)
+    fd = np.fft.fftfreq(dspad.shape[0], t[1] - t[0]).to(u.mHz)
+
+    conspec = np.fft.fftshift(conspec)
+    tau = np.fft.fftshift(tau) << tau.unit
```

Comment:
The `<< tau.unit` shouldn't be necessary any more (it was for astropy < 4.0); same for next line

Reply:
Removed